site/src/modules/provisioners/ProvisionerAlert.stories.tsx

```diff
@@ -21,6 +21,26 @@ export default meta;
 type Story = StoryObj<typeof ProvisionerAlert>;
 
 export const Info: Story = {};
+
+export const InfoStyledForLogs: Story = {
+	args: {
+		variant: AlertVariant.InLogs,
```

Comment:
My main question is about naming - does "InLogs" reflect variant's behavior or location? Maybe we should consider a bit different name, something that is not referring to the place where is used but rather its function (no frame?)

Reply:
updated